package.json

```diff
@@ -193,6 +193,12 @@
           ],
           "scope": "machine",
           "type": "string"
+        },
+        "ruff.enableExperimentalFormatter": {
+          "default": false,
+          "markdownDescription": "Controls whether Ruff registers as capable of code formatting.",
```

Comment:
I think we should be more explicit about what experimental means in this context and whether toggling the setting requires restarting VS code.